pennylane_qsharp/device.py

```diff
@@ -139,7 +138,7 @@ def post_apply(self): #pragma no cover
             device_wires = self.map_wires(e.wires)
 
             self.measure += "set resultArray w/= {wires[0]} <- ".format(wires=device_wires.tolist())
-            self.measure += self._observable_map[e.name].format(p=e.parameters, wires=device_wires.tolist())
```

Comment:
I think the braket bug came from the fact that we pass two objects here, but the observable strings do not have "p"/parameters.

Reply:
Oh good catch!